packages/ibm-products/src/components/Datagrid/Datagrid.stories.js

```diff
@@ -511,7 +511,6 @@ export const BatchActions = () => {
         id: 'delete',
         itemText: 'Delete',
         icon: TrashCan,
-        isDelete: true,
```

Comment:
Removed.

When rendering _one or two_ "row actions", `useActionsColumn` will render `OverflowMenu` "trigger" components, without menu items, as buttons. (They should render `IconButtons` instead, but that's another story). `OverflowMenu` "trigger" components do not accept `isDelete` as props.

When rendering _more than two_ "row actions", `useActionsColumn` will render one `OverflowMenu` trigger and then one menu item per action. Each item may accept `isDelete` as a prop to render an item red when hovered over.

Reply:
I've created https://github.com/carbon-design-system/ibm-products/issues/3662 so that we can still support `isDelete`. This could also potentially include some refactoring to use `IconButton`s in the case of 1 or 2 actions.